talon/signature/bruteforce.py

```diff
@@ -166,6 +168,9 @@ def _mark_candidate_indexes(lines, candidate):
     # mark lines starting from bottom up
     for i, line_idx in reversed(list(enumerate(candidate))):
         if len(lines[line_idx].strip()) > TOO_LONG_SIGNATURE_LINE:
+            # if the line contains a url, ignore the length requirement
+            if RE_URL.search(lines[line_idx]):
+                continue
```

Comment:
The line could still start with a dash, in which case it should be marked as "d".